nilearn/_utils/estimator_checks.py

```diff
@@ -978,12 +978,8 @@ def check_img_estimator_doc_attributes(estimator) -> None:
     documented_parameters = {
         param.name: param.type for param in doc["Parameters"]
     }
-    # TODO (nilearn >= 0.13.0)
-    # remove the 'and param != "clean_kwargs"'
     undocumented_parameters = [
-        param
-        for param in parameters
-        if param not in documented_parameters and param != "clean_kwargs"
```

Comment:
this may trigger a few tests to fail: in that case the doc strings of some nifti masker may need to be adapted